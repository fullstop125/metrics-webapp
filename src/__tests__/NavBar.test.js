import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../__Mocks__/NavBar';

it('renders correctly', () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
