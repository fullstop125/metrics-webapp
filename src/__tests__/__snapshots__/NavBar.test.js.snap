// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <ul
    className="nav-ul"
  >
    <li>
      pathname === '/' ? ' ' : 'details'
    </li>
    <li>
      pathname === '/' ? 'Company Information' : 'Details'
    </li>
  </ul>
</nav>
`;
